### PR TITLE
[Shell] Update shell messages

### LINKED ISSUE #30919: [Shell] Improve the help text for the Shell
The current output for `/help` is the following:
```
| Ballerina Shell Help
| 
| Type a Ballerina language expression, statement, or declaration.
| Or type one of the following commands:
| 
| /exit           - Exit the terminal shell.
| /debug          - Toggle debug mode. You will see debug messages.
| /reset          - Reset REPL state.
|                   All previous code snippets will be removed from REPL memory.
| /file <FILE>    - Load declarations defined in <FILE>. e.g., /file my_file.bal
| /remove <NAMES> - Delete declarations specified by <NAMES>. e.g., /remove function1 var1 var2
| /imports        - List the available imports.
| /vars           - List the declared variables and their values.
| /dclns          - List the user defined module level declarations.
| /help           - Get this message.
| /help <TOPIC>   - Get help on <TOPIC>. eg: /help var
| /help topics    - See all available topic names.
```
There are a couple of minor improvements to be done here.
- Should this show which version of Ballerina this is?
- "Exit the Shell."
- "Toggle the debug mode ON/OFF."
- "Delete the declarations..."
- for `/help` => "Displays the help text"

### LINKED ISSUE #30921: [Shell] Improve error message given when loading from a file fails
See the following:
```
=$ /file q                                                                                                                                                                                                         
| Error loading from file: q
```
If a more specific error cannot be given, shall we change this to say `Failed to load declarations from file: q`

### PR DESCRIPTION
**Pull request by DamithaSenevirathne**
Fixes #30921
Fixes #30919 

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: ballerina-shell/modules/shell-cli/src/main/resources/command.header.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Welcome to Ballerina Shell REPL.
+Ballerina version : Ballerina Swan Lake Beta 1
```
Comment (pubudu91) [inline ballerina-shell/modules/shell-cli/src/main/resources/command.header.txt:2]: This shouldn't be hard coded

Comment (DamithaSenevirathne) [reply ballerina-shell/modules/shell-cli/src/main/resources/command.header.txt]: Removed in ee61c122de135136141783b29a433deeabc874c3

File: ballerina-shell/modules/shell-cli/src/main/resources/command.help.txt [hunk 2]
```diff
@@ -3,15 +3,15 @@ Ballerina Shell Help
 Type a Ballerina language expression, statement, or declaration.
 Or type one of the following commands:
 
-/exit           - Exit the terminal shell.
-/debug          - Toggle debug mode. You will see debug messages.
+/exit           - Exit the Shell.
```
Comment (IMS94) [inline ballerina-shell/modules/shell-cli/src/main/resources/command.help.txt:6]:
```suggestion
/exit           - Exit the shell.
```

**Review by IMS94 — DISMISSED** [review]
Approved with a small suggestion.

**Review by pubudu91 — APPROVED** [review]

**Pull request merged**